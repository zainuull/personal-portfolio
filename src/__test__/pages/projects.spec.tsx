import ProjectsPage from '@/pages/projects';
import { render } from '@testing-library/react';

describe('projects page', () => {
  it('render projects page', () => {
    const page = render(<ProjectsPage />);
    expect(page).toMatchSnapshot();
  });
});
