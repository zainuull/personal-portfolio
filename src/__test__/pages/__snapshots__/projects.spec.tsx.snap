// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`projects page render projects page 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="w-full min-h-screen overflow-hidden bg-primary pb-10"
      >
        <div
          class="w-full h-full flex flex-col items-center gap-x-4 text-white mt-24"
        >
          <h1
            class="text-4xl font-semibold border-b-2 border-secondary"
          >
            Projects
          </h1>
          <div
            class="fw-full flex flex-col mt-8 px-4 gap-y-8"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="w-full min-h-screen overflow-hidden bg-primary pb-10"
    >
      <div
        class="w-full h-full flex flex-col items-center gap-x-4 text-white mt-24"
      >
        <h1
          class="text-4xl font-semibold border-b-2 border-secondary"
        >
          Projects
        </h1>
        <div
          class="fw-full flex flex-col mt-8 px-4 gap-y-8"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
